lib/v8.js

```diff
@@ -397,21 +402,74 @@ function deserialize(buffer) {
   return der.readValue();
 }
 
+const GC_PROFILER_DATA_TYPE = {
```

Comment:
Can we pass the constants from the bindings instead of hard-coding them here?

Reply:
Done !